python/pylibcudf/pylibcudf/reshape.pyx

```diff
@@ -67,3 +76,41 @@ cpdef Table tile(Table source_table, size_type count):
         c_result = cpp_tile(source_table.view(), count)
 
     return Table.from_libcudf(move(c_result))
+
+
+cpdef void table_to_array(
+    Table input_table,
+    uintptr_t ptr,
+    size_type size,
+    Stream stream=None
+):
+    """
+    Copy a table into a preallocated column-major device array.
+    Parameters
```

Comment:
A blank line between sections is needed for this to format correctly in the docs.
```suggestion

    Parameters
```